cds-snc/platform-forms-client
# Allow copy/paste multiple confirmation codes

# Summary | Résumé

Updates the `Confirm receipt of responses` dialog to allow pasting a column of confirmation_codes from a csv or spreadsheet.

Will validate and exclude any non-confirmation_code entries (ie, header row `confirmation_code` or blank rows)

### To test
- Export a csv from the API, or use this [example.csv](https://github.com/cds-snc/platform-forms-client/files/13299287/example.csv) (it doesn't matter if the confirmation codes are invalid for the form)
- Open the csv in Google Sheets or Excel.
- Click on the column header for `confirmation_code` to select the column and CMD-C or CTRL-C to copy to clipboard
- Open the `Confirm receipt of responses` dialog from the Responses page of a form that has responses.
- Click into the input and paste the data
- You should see the confirmation codes entered, with no extra empty rows or header string
